### ISSUE TITLE
MetroProgressBar orientation does not work

### ISSUE THREAD

**Issue by widgetqueen** [MahApps/MahApps.Metro#2148]
Good afternoon!

The Orientation field on the MetroProgressBar control does not seem to work - I tried the workaround suggested here: https://github.com/MahApps/MahApps.Metro/issues/1465 but it does not work for my purposes. Rotating the progress bar creates issues within my grid/viewbox structure that I don't have when simply using a vertically oriented progress bar.

**Comment by punker76** [comment]
@widgetqueen what should be happen if `IsIndeterminate="True"`? should it be come from down to up o from up to down?

**Comment by widgetqueen** [comment]
I'm not sure I understand your question. When I set IsIndeterminate="True", the progress bar disappears. Currently, when I set IsIndeterminate to false and the orientation to vertical, the progress bar does not fill in correctly according to its value. The progress bar itself appears to be vertical, but when the fill is applied, it still appears to fill in horizontally.

**Comment by petvetbr** [comment]
I'm taking a look at this issue.

**Comment by realies** [comment]
This is still not working on the latest alpha. Current solution:

```
<Style TargetType="Controls:MetroProgressBar">
 <Setter Property="LayoutTransform">
  <Setter.Value>
   <RotateTransform CenterX="0.5" CenterY="0.5" Angle="-90"/>
  </Setter.Value>
 </Setter>
</Style>
```

**Comment by punker76** [comment]
@widgetqueen @petvetbr fixed with d4aa412870a9ae0898a39813a71aad87f572c560
